kolibri/core/assets/src/views/CoreMenu/CoreMenuOption.vue

```diff
@@ -37,7 +37,8 @@
     props: {
       label: {
         type: String,
-        required: true,
+        required: false,
```

Comment:
Maybe just for safety, we should conditionalize the `KIcon` component on `icon` being truthy, and the div for `{{ label }}` on the label being truthy?

Reply:
Yes that's a good point, will do